README.md

```diff
@@ -107,6 +107,11 @@ if you want to use just the `foundry.toml` file.
 
 ### DEPENDENCY INSTALLATION
 
+Initialize a fresh installation by using
+```bash
+soldeer init
+```
```

Comment:
worth adding a bit more context on what this is actually doing, deleting libs/gitsubmodules.